style.css

```diff
@@ -68,3 +77,253 @@ body {
+  font-size: 15px;
+  width: auto;
+  border: 1px solid #8993a4;
+  height: 20px;
+  color: #091e42;
+  padding: 10px 12px 10px 12px;
+}
+
+.project-1-btn {
+  height: 48px;
+  width: 124px;
+  background-color: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-size: 17px;
+  color: #fff;
+}
+
+.project-x {
+  display: flex;
+  flex-direction: column;
+  gap: 12px;
+  background: url("../images/Mask Group.svg") 50% 50% /cover no-repeat;
+  font-family: 'Inter', sans-serif;
+}
+
+.project-x-title {
+  font-size: 2rem;
+  padding-left: 24px;
+  color: #fff;
+}
+
+.project-x-para {
+  font-family: 'Inter', sans-serif;
+  font-size: 1rem;
+  padding-left: 24px;
+  padding-right: 24px;
+  color: #fff;
+}
+
+.project-x-list {
+  display: flex;
+  gap: 24px;
+  height: 40px;
+  align-items: center;
+}
+
+.proj-list {
+  height: 20px;
+  width: auto;
+  background-color: rgba(255, 255, 255, 0.24);
+  padding: 10px 12px 10px 12px;
+  font-family: 'Inter', sans-serif;
+  font-size: 15px;
+  font-weight: medium;
+  color: #fff;
+}
+
+.project-btn {
+  height: 48px;
+  width: auto;
+  background-color: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-weight: bold;
+  font-size: 17px;
+  color: #fff;
+}
+
+.about-me {
+  background-color: #1c1a19;
+  height: auto;
+  padding: 120px 24px 120px 24px;
+}
+
+.about-container {
+  display: grid;
+  grid-template-columns: 1;
+  grid-template-rows: auto;
+  padding: 120px 24px 120px 24px;
+  grid-gap: 24px;
+  align-items: center;
+}
+
+.about-me-1 {
+  background: url("../images/Frame.svg") bottom right  no-repeat;
+  height: 563px;
+  display: flex;
+  flex-direction: column;
+  gap: 24px;
+}
+
+.about-me-title {
+  height: 79px;
+  font-family: 'Crete Round', serif;
+  font-size: 72px;
+  color: #ff6b00;
+  margin-top: 0;
+}
+
+.about-me-para {
+  font-family: 'Inter', sans-serif;
+  color: #f4f5f7;
+  font-weight: 400;
+  font-size: 1rem;
+  height: 148px;
+}
+
+.about-me-btn {
+  height: 48px;
+  width: 161px;
+  background-color: #ff6b00;
+  font-family: 'Inter', sans-serif;
+  font-weight: bold;
+  font-size: 17px;
+  color: #fff;
+  margin-left: 0;
+}
+
+.line-container {
+  height: 17px;
+  width: 327px;
+  margin: auto;
+}
+
+.line-1 {
+  border: 1px solid #fff;
+  height: 0;
+}
+
+.about-me-common {
+  font-family: 'Inter', sans-serif;
+  color: #fff;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 120px 24px;
+  gap: 24px;
+}
+
+.about-start {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  gap: 24px;
+}
+
+.about-title {
+  font-family: 'Crete Round', serif;
+  font-size: 32px;
+}
+
+.about-my-list {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  height: 216px;
+  margin: auto;
+}
+
+.about-list {
+  height: 48px;
+  width: auto;
+  font-size: 22px;
+}
+
+.button:hover {
+  background-color: #fff5e1;
+  color: #f55800;
+}
+
+/* .button:pressed {
+  background-color: #e05304;
+  color: #f4f1f0;
+} */
```

Comment:
- [ ] Please remove this commented code because leaving commented code in your code base is not a good practice and makes your code look cluttered.👌😊